android/src/main/java/com/mybigday/rnmediameta/RNMediaMeta.java

```diff
@@ -78,7 +79,18 @@ private void putString(WritableMap map, String key, String value) {
     if (value != null) map.putString(key, value);
   }
 
-  private void getMetadata(String path, Promise promise) {
+  private void getMetadata(String pathOrJSONString, Promise promise) {
```

Comment:
I started to go that route. I realized, I didn't want to change the ```get``` method as I was hoping not to break existing users, but if that's what you want I'll do it.